lib/app_header.dart

```diff
@@ -0,0 +1,35 @@
+import 'package:flutter/material.dart';
+
+class AppHeader extends StatelessWidget {
+  AppHeader({super.key});
+
+  String appTitle = 'The Movie DB';
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      color: const Color(0xff22303C),
+      padding: const EdgeInsets.all(8),
+      child: Row(
+        children: [
+          const Icon(
+            Icons.menu,
+            color: Color(0xffFAFAFA),
+            size: 35,
+          ),
+          Padding(
+            padding: const EdgeInsets.fromLTRB(18, 0, 0, 0),
```

Comment:
LRTB defaults value is 0 , so you change this to EdgeInsets.only(left: 18.0) and also constant for the value